src/utils/calendar.ts

```diff
@@ -0,0 +1,119 @@
+import { config } from '@/config.js';
+import { VEvent, fromURL } from 'node-ical';
+import * as dateUtils from './date.js';
+
+// Interface pour affichés les cours avec seulement les informations nécessaires
+interface Lesson {
+  name: string;
+  start: string;
+  end: string;
+  room: string;
+  teacher: string;
+}
+
+// Les données du fichier ICS
+const calendarData = await fromURL(config.SCHEDULE_URL);
+
+const today = dateUtils.startOfToday(); // Date d'aujourd'hui       Luxon : DateTime.now().startOf('day')
+const startOfWeek = dateUtils.startOfCurrentWeek(); // Date du début de la semaine    Luxon : DateTime.now().startOf('week')
+const endOfWeek = dateUtils.endOfCurrentWeek(); // Date de fin de la semaine   Luxon : startOfWeek.plus({ days: 4 }).endOf('day')
+
+/*
+ * Crée un tableau de cours à partir des données du fichier ICS
+ */
+function createLessonsFromData(data: VEvent[]): Lesson[] {
+  return data.map((lesson) => {
+    // Convertir les dates en version lisible
+    const start_time = dateUtils.humanDate(new Date(lesson.start)); // Luxon : DateTime.fromJSDate(lesson.start).toLocaleString(DateTime.DATE_MED_WITH_WEEKDAY, {locale : 'fr'});
+    const end_time = dateUtils.humanDate(new Date(lesson.end));
+
+    return {
+      name: lesson.summary,
+      start: start_time,
+      end: end_time,
+      room: lesson.location.replace('Remicourt_', ''), // Balek du Remicourt
+      teacher: lesson.description.split('\n')[2], // TODO: Trouver un moyen de récupérer le nom du prof
+    };
+  });
+}
+
+/*
+ * Récupère les cours du jour
+ */
+export function getTodayClasses(): Lesson[] {
```

Comment:
```suggestion
export function getTodaysClasses(): Lesson[] {
```
Parce que "get today's classes"